### ISSUE TITLE
Overlap of Express checkout buttons in cart

### ISSUE THREAD

**Issue by PanosSynetos** [woocommerce/woocommerce-gutenberg-products-block#10793]
I noticed that when having 2 or 3 express checkout buttons there is an overlap in Block Cart

![image](https://github.com/woocommerce/woocommerce-blocks/assets/2484390/8f80c36b-98af-4a26-9f04-506f2f75ad89)

![image](https://github.com/woocommerce/woocommerce-blocks/assets/2484390/2b865586-a379-4762-91b4-ae1a1566615a)

If the sidebar is wider, it works fine, changed it to 50-50 with the inspector
![image](https://github.com/woocommerce/woocommerce-blocks/assets/2484390/76162561-5192-4f9f-bdc7-364bdda27bcb)

## Checkout looks fine

![image](https://github.com/woocommerce/woocommerce-blocks/assets/2484390/6a715224-6de4-47dd-ba69-c1f0dc2e7203)

# System Status Report

<details><summary>SSR</summary>
```
`
### WordPress Environment ###

WordPress address (URL): https://devtrialpanos.wpcomstaging.com
Site address (URL): https://devtrialpanos.wpcomstaging.com
WC Version: 8.0.3
REST API Version: ✔ 8.0.3
WC Blocks Version: ✔ 10.6.6
Action Scheduler Version: ✔ 3.6.1
Log Directory Writable: ✔
WP Version: ❌ 6.2.1 - There is a newer version of WordPress available (6.3.1)
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.0.30
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 8.0.1
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 10.6.13-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.0.3
WC Database Prefix: wp_
Total Database Size: 21.37MB
Database Data Size: 14.90MB
Database Index Size: 6.47MB
wp_woocommerce_sessions: Data: 3.52MB + Index: 0.14MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.19MB + Index: 0.14MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.11MB + Index: 0.09MB + Engine InnoDB
wp_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_automatewoo_customers: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_guests: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_log_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_gla_attribute_mapping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_budget_recommendations: Data: 0.22MB + Index: 0.14MB + Engine InnoDB
wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 2.36MB + Index: 0.33MB + Engine InnoDB
wp_postmeta: Data: 0.09MB + Index: 0.03MB + Engine InnoDB
wp_posts: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_wc_orders_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_addresses: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_bundle_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_operational_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_bis_activity: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_woocommerce_bis_notifications: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_woocommerce_bis_notificationsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_prl_deploymentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_prl_deployments: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_prl_frequencies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_prl_tracking_conversions: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_actionscheduler_actions: Data: 0.11MB + Index: 0.11MB + Engine InnoDB
__wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_actionscheduler_logs: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
__wp_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
__wp_automatewoo_customers: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_automatewoo_events: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_automatewoo_guests: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_automatewoo_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_automatewoo_log_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_gla_attribute_mapping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_gla_budget_recommendations: Data: 0.20MB + Index: 0.11MB + Engine InnoDB
__wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_options: Data: 2.34MB + Index: 0.25MB + Engine InnoDB
__wp_postmeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
__wp_posts: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
__wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_usermeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
__wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
__wp_wc_admin_note_actions: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
__wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_order_bundle_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_bis_activity: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_woocommerce_bis_notifications: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_woocommerce_bis_notificationsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_prl_deploymentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_prl_deployments: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_prl_frequencies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_prl_tracking_conversions: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_woocommerce_sessions: Data: 2.52MB + Index: 0.09MB + Engine InnoDB
__wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 17
customize_changeset: 14
jetpack_migration: 2
page: 14
post: 3
product: 5
revision: 10
shop_order: 3
wp_global_styles: 3
wp_navigation: 2
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (28) ###

Akismet Anti-Spam: Spam Protection: by Automattic - Anti Spam Team – 5.2
AutomateWoo: by WooCommerce – 6.0.3
Facebook for WooCommerce: by Facebook – 3.0.28
WordPress.com Editing Toolkit: by Automattic – 3.72117
Google Listings and Ads: by WooCommerce – 2.4.11
Gutenberg: by Gutenberg Team – 16.2.1
Jetpack: by Automattic – 12.3
Layout Grid: by Automattic – 1.8.4
Pinterest for WooCommerce: by WooCommerce – 1.3.5 (update to version 1.3.8 is available)
TikTok: by TikTok – 1.0.15
Woo AI: by WooCommerce – 0.3.0
WooCommerce Back In Stock Notifications: by WooCommerce – 1.4.3 (update to version 1.6.3 is available)
WooCommerce Brands: by WooCommerce – 1.6.43 (update to version 1.6.56 is available)
WooCommerce Stripe Gateway: by WooCommerce – 7.5.0
WooCommerce Gift Cards: by WooCommerce – 1.15.1 (update to version 1.16.4 is available)
WooCommerce Google Analytics Integration: by WooCommerce – 1.8.3
WooCommerce Min/Max Quantities: by WooCommerce – 4.0.6 (update to version 4.1.1 is available)
WooCommerce Payments Dev Tools: by Automattic –
WooCommerce Payments: by Automattic – 6.3.2
WooCommerce PayPal Payments: by WooCommerce – 2.2.0 (update to version 2.2.2 is available)
WooCommerce Product Add-ons: by WooCommerce – 5.0.3 (update to version 6.4.5 is available)
WooCommerce Product Bundles: by WooCommerce – 6.17.4 (update to version 6.22.3 is available)
WooCommerce Product Recommendations: by WooCommerce – 2.2.1 (update to version 2.4.2 is available)
WooCommerce Shipping & Tax: by WooCommerce – 2.3.0 (update to version 2.3.3 is available)
WooCommerce Shipment Tracking: by WooCommerce – 2.0.0 (update to version 2.3.5 is available)
WooCommerce: by Automattic – 8.0.3

### Inactive Plugins (15) ###

Avalara AvaTax: by Avalara – 2.6.0
Classic Editor: by WordPress Contributors – 1.6.3
Crowdsignal Forms: by Automattic – 1.7.0
Crowdsignal Polls & Ratings: by Automattic
Inc. – 3.0.10

Page Optimize: by Automattic – 0.5.3
TaxJar - Sales Tax Automation for WooCommerce: by TaxJar – 4.1.5
WooCommerce Australia Post Shipping: by WooCommerce – 2.5.0 (update to version 2.5.4 is available)
WooCommerce Blocks: by Automattic – 10.6.0
WooCommerce Canada Post Shipping: by WooCommerce – 2.7.1 (update to version 2.8.1 is available)
WooCommerce EU VAT Number: by WooCommerce – 2.8.7
WooCommerce FedEx Shipping: by WooCommerce – 3.8.3
WooCommerce PayPal Payments: by WooCommerce – 2.1.0 (update to version 2.2.2 is available)
WooCommerce Royal Mail: by WooCommerce – 2.7.1 (update to version 2.9.0 is available)
WooCommerce UPS Shipping: by WooCommerce – 3.4.1 (update to version 3.5.4 is available)
WooCommerce USPS Shipping: by WooCommerce – 4.6.2 (update to version 4.7.3 is available)

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Must Use Plugins (1) ###

WP.com Site Helper: by  –

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #87 - /shop/
Cart: #88 - /cart/
Checkout: #89 - /checkout/
My account: #90 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Tsubaki
Version: 1.1.4
Author URL: https://automattic.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: /wordpress/plugins/woocommerce/8.0.3/packages/woocommerce-blocks/templates/notices/error.php
/wordpress/plugins/woocommerce/8.0.3/packages/woocommerce-blocks/templates/notices/notice.php
/wordpress/plugins/woocommerce/8.0.3/packages/woocommerce-blocks/templates/notices/success.php


### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://devtrialpanos.wpcomstaging.com
Subscriptions-core Library Version: 6.0.0
Subscription Statuses: –
WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No

### Store Setup ###

Country / State: United States (US) — California

### Subscriptions by Payment Gateway ###


### Payment Gateway Support ###

WooPayments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
gateway_scheduled_payments
tokenization
add_payment_method

Direct bank transfer: products
Check payments: products
Cash on delivery: products
PayPal: refunds
products


### Back In Stock ###

Database Version: 1.4.3
Loopback Test: ✔

### Gift Cards ###

Database Version: 1.15.1
Loopback Test: ✔
Task Queueing Test: ✔

### Product Bundles ###

Database Version: 6.17.4
Loopback Test: ✔
Template Overrides: –

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: US
WooCommerce currency supported: ✔
Advanced Card Processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: –
PayPal Vault enabled: –
ACDC Vault enabled: –
Logging enabled: –
Reference Transactions: ✔
Used PayPal Checkout plugin: –
Tracking enabled: ✔

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
navigation
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-02-17 13:23:30 +02:00
Options: ✔
Notes: 81
Onboarding: skipped

### WooCommerce Payments ###

Version: 6.3.2
Connected to WPCOM: Yes
Blog ID: 215607453
Account ID: acct_1MdaFXFyyZ3YxK1J

### Action Scheduler ###

Canceled: 5
Oldest: 2023-08-01 08:32:13 +0300
Newest: 2023-08-29 14:02:46 +0300

Complete: 335
Oldest: 2023-08-01 08:32:13 +0300
Newest: 2023-08-31 13:02:06 +0300

Failed: 3
Oldest: 2023-02-15 03:28:00 +0200
Newest: 2023-05-18 12:13:05 +0300

Pending: 17
Oldest: 2023-08-31 13:03:17 +0300
Newest: 2023-09-01 12:48:17 +0300


### Product Recommendations ###

Database Version: 2.2.1
Loopback Test: ✔
Task Queueing Test: ✔
Page Cache Test: No cache detected

### Status report information ###

Generated at: 2023-08-31 13:03:48 +03:00
`
```
</details>

**Comment by ralucaStan** [comment]
Hey @PanosSynetos , thank you for opening this. 
I recently fixed this in https://github.com/woocommerce/woocommerce-blocks/pull/10534; I checked with latest trunk and could not replicate.

<img width="646" alt="Screenshot 2023-08-31 at 13 14 47" src="https://github.com/woocommerce/woocommerce-blocks/assets/1628454/4e1ad3a8-d5c2-46a5-bb8b-ad8240843433">

**Comment by ralucaStan** [comment]
I'll close this issue. The fix is part of WooCommerce Blocks 10.9.0 which will be released with WC 8.1
